src/app/visitor/account/tokens.html

```diff
@@ -0,0 +1,232 @@
+            </a>
+            <div class="clearfix"></div>
+            <!-- tokens info table -->
+            <div class="panel panel-default">
+
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <td>
+                                Credit &amp; Debit Cards
+                            </td>
+                            <td></td>
+                            <td class="text-center">
+                                Expires
+                            </td>
+                            <td></td>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-repeat="token in tokens">
+                            <td>{{token.Extra.type}} ending in {{token.Extra.number}}</td>
+                            <td><span ng-if="showTokenMethod">{{token.Desc}}</span></td>
+                            <td class="text-center">{{parseDate(token.Extra.expiration_date)}}</td>
+                            <td class="text-right">
+                                <button class="btn btn-primary btn-block-xs"
+                                    ng-click="remove(token.ID)">
+                                    <i class="fa fa-remove fa-fw"></i>
+                                </button>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+            </div>
+        </div>
+    </div>
+
+    <!-- add token modal -->
+    <div class="modal fade"
+        id="parent_popup_token"
+        tabindex="-1"
+        role="dialog">
+        <div class="modal-dialog modal-md">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button data-dismiss="modal"
+                        class="close"
+                        type="button">
+                        <span aria-hidden="true">×</span><span class="sr-only">Close</span>
+                    </button>
+                    <h4 class="modal-title">Add Credit Card</h4>
+                </div>
+                <div class="modal-body">
+
+                    <!-- note: this is also outside the modal -->
+                    <!-- error message -->
+                    <ot-message-manager item="message"></ot-message-manager>
+
+                    <div class="radio-subform paypal">
+                        <form class="form-cc"
+                              name="creditCardConroller"
+                              novalidate
+                              ng-submit="save()">
+
+                            <div class="form-group form-cc__able-methods" ng-if="showTokenMethod">
+                                <div ng-repeat="tokenAbleMethod in tokenAbleMethods">
+                                    <label>
+                                        <input type="radio"
+                                               ng-model="creditCardData.payment"
```

Comment:
@igorGalchenko please change `creditCardData.payment` to creditCardData.payment_method`